web/text_layer_builder.css

```diff
@@ -24,7 +24,7 @@
   line-height: 1;
 }
 
-.textLayer > span {
+.textLayer span {
```

Comment:
If we change this, it should probably also be changed here since that file is based on this one: https://github.com/mozilla/pdf.js/blob/master/test/text_layer_test.css#L26

Reply:
Additionally, there's a handful of `.textLayer > span`-related rules in https://github.com/mozilla/pdf.js/blob/master/web/viewer.css so please check if those also need to be updated.

Reply:
Not sure how I missed those. Technically one or two can be left since the property is inherited, but for consistency it's probably better to just change them all.